src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -117,7 +122,8 @@
         "Project ID is required to access the auth service. Use a service account credential or "
             + "set the project ID explicitly via FirebaseOptions. Alternatively you can also "
             + "set the project ID via the GOOGLE_CLOUD_PROJECT environment variable.");
-    this.baseUrl = String.format(ID_TOOLKIT_URL, projectId);
+    this.userBaseUrl = String.format(ID_TOOLKIT_URL, "v1", projectId);
```

Comment:
Nit: Lets call them `userMgtBaseUrl` and `tenantMgtBaseUrl` for clarity.

Reply:
Done.